V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt

```diff
@@ -417,6 +417,7 @@ class MainActivity : BaseActivity(), NavigationView.OnNavigationItemSelectedList
                         URL(url).readText()
                     } catch (e: Exception) {
                         e.printStackTrace()
+                        return@launch
                         ""
```

Comment:
Thanks for your pr. Looks like after your return statement, the "" in line 421 is useless and should be cleaned up

Reply:
OK，Clean now